### PR TITLE
Change type for OPTIMISM

### PR DESCRIPTION
**Pull request by cryptomanz1**

### CODE REVIEW

File: blockchains/optimism/assets/0xffD27b572246f35c992EfB28DD8cB730b93Ed301/info.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "Rubicon",
     "symbol": "Rubicon",
-    "type": "OPTIMISM",
```
Comment (Kimj2539) [inline blockchains/optimism/assets/0xffD27b572246f35c992EfB28DD8cB730b93Ed301/info.json]: _opmainnet
_

File: blockchains/optimism/assets/0xCA7661Ba64A7667E3006c421C181502d545D1911/info.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "ARAW",
     "symbol": "ARAW",
-    "type": "OPTIMISM",
```
Comment (Kimj2539) [inline blockchains/optimism/assets/0xCA7661Ba64A7667E3006c421C181502d545D1911/info.json]: ``

File: blockchains/optimism/assets/0xffD27b572246f35c992EfB28DD8cB730b93Ed301/info.json
Comment (Kimj2539) [inline blockchains/optimism/assets/0xffD27b572246f35c992EfB28DD8cB730b93Ed301/info.json:1]: #25547

File: blockchains/optimism/assets/0x50Bce64397C75488465253c0A034b8097FeA6578/info.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
 {
     "name": "HanChain",
-    "type": "OPTIMISM",
```
Comment (iCell) [inline blockchains/optimism/assets/0x50Bce64397C75488465253c0A034b8097FeA6578/info.json:3]: What's the differences between these changes, the `type` are still `OPTIMISM`, you just changed the ordering?

Comment (cryptomanz1) [reply blockchains/optimism/assets/0x50Bce64397C75488465253c0A034b8097FeA6578/info.json:3]: I've changed the `type` from `OPTIMISM` to `OPMAINNET`, but the `merge-fee-bot` automatically fixed my changes. Could you please fix it?
https://github.com/trustwallet/assets/pull/25547/commits/e9f090c4b663e22e677f7821b40ad97682a19dc1

**Pull request closed without merging**